tools/convertTranslations.js

```diff
@@ -189,7 +189,7 @@ function toLocalizableStrings(lines) {
       if (key.startsWith("INFOPLIST.")) {
         continue;
       }
-      let value = line[1].replace(/\\/g, '\\\\').replace(/\n/g, '\\n').replace(/\r/g, '\\r').replace(/\t/g, '\\t').replace(/"/g, '\\"');
+      let value = line[1].replace(/\\/g, '\\').replace(/\n/g, '\n').replace(/\r/g, '\r').replace(/\t/g, '\t').replace(/"/g, '\"');
```

Comment:
terms like `replace(/\\/g, '\\')` or `replace(/\n/g, '\n')` have no effect. `\\` will be replaced by `\\` and `\n` by `\n`.
That's why this line can be reduced to `let value = line[1];` You can also substitute `value` by `line[1]` on line 193

Reply:
To be clear: the original code `replace(/\\/g, '\\\\')` was wrong! So good catch here!